src/main/drivers/dshot_bitbang.c

```diff
@@ -475,8 +476,12 @@ static void bbWriteInt(uint8_t motorIndex, uint16_t value)
         return;
     }
 
-    // If there is a command ready to go overwrite the value and send that instead
+    // fetch requestTelemetry from motors. Needs to be refactored.
+    motorDmaOutput_t * const motor = getMotorDmaOutput(motorIndex);
```

Comment:
This `const` makes no sense.

Reply:
Why not? I just mirrored the bbMotor* const declaration a few lines up. This is a const pointer to a mutable object.